docs/app-dev/ecosystem.json

```diff
@@ -133,6 +133,12 @@
       "language": "Javascript",
       "author": "Tendermint"
     },
+    {
+      "name": "rust-tsp",
```

Comment:
Shouldn't this be renamed to rust-abci (the repo, too)?

Reply:
Probably, but I don't have rights to do so.

Reply:
Done!